.gitignore

```diff
@@ -513,6 +513,7 @@ PYPI_VERSION
 # docs builds
 docs/make.bat
 docs/Makefile
+docs/jupyter_execute/
```

Comment:
I mentioned command to build as `sphinx-build -b html . _build`. My bad. 
I see using `sphinx-build -b html . _build/html` is more appropriate in which case the jupyter_execute ends up inside _build.
```suggestion
```

Reply:
reverting revert. no need to be bothered with details